Jenkinsfile

```diff
@@ -94,11 +97,12 @@ builds.ath = {
             runATH jenkins: fileUri, metadataFile: metadataPath
         }
     }
-}
+}*/
 
 builds.failFast = failFast
 parallel builds
-infra.maybePublishIncrementals()
+// TODO: Restore Incrementals once https://groups.google.com/forum/#!topic/jenkinsci-dev/v9d-XosOp2s is resolved
+// infra.maybePublishIncrementals()
```

Comment:
Fine as a hotfix, but actually what we should do _anyway_ (permanently) is switch this method to use an ACI agent.

Reply:
Do not disagree, should be done in https://github.com/jenkins-infra/pipeline-library

Reply:
I will prepare a patch for it.

Reply:
https://github.com/jenkins-infra/pipeline-library/pull/107

Reply:
Merged, so I think this hunk could now be reverted.